src/features/publicOffer/components/PublicOfferParticle.tsx

```
@@ -0,0 +1,59 @@
+                  return (
+                    <li key={index} className="flex flex-col">
+                      <div className="flex gap-1">
+                        <span className=" font-medium">{index + 1}.</span>
+                        <p>{item.text}</p>
+                      </div>
+
+                      {item.subList && (
+                        <ul className="pl-7">
+                          {item.subList.map(subList => {
+                            return (
+                              <li className="flex gap-1">
```

Biome (2.1.2)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)